wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,73 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I restricted?
+A user may get restricted if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
```

Comment:
combine with L7?

Reply:
combine what?

Reply:
`Using multiple accounts.` with `Circumventing a punishment by making a new account.`?

Reply:
these are different though

Reply:
ignore this review point

Reply:
lol ok